be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -156,7 +156,9 @@ Status SegmentIterator::_init(bool is_vec) {
     // Remove rows that have been marked deleted
     if (_opts.delete_bitmap.count(segment_id()) > 0 &&
         _opts.delete_bitmap[segment_id()] != nullptr) {
+        size_t pre_size = _row_bitmap.cardinality();
         _row_bitmap -= *(_opts.delete_bitmap[segment_id()]);
```

Comment:
Maybe we can add a CHECK `pre_size > _row_bitmap.cardinality()`

Reply:
Why do we need such a CHECK here? Do you want to ensure that the delete_bitmap is not empty?

Reply:
Sorry,  I read it wrong, I read `-=` as `=` 😥

Reply:
LOL, thanks for your review~